integration_tests/src/main/python/spark_session.py

```diff
@@ -111,6 +111,10 @@ def is_before_spark_320():
 def is_before_spark_330():
     return spark_version() < "3.3.0"
 
+# The bug SPARK-37451 only affects the following versions
+def is_neg_dec_scale_bug_version():
+    return ("3.1.1" <= spark_version() < "3.1.3") or ("3.2.0" <= spark_version() < "3.2.1")
```

Comment:
the last part can just be spark_version == 3.2.0

Reply:
What if we have `3.2.0-cdh` or whatever?

Reply:
ah that is a good point, could have a minor version past the 3.2.0, so leave it as you have it.